File: 042/a.py
# includeがうまく表示されないので，カッコなしで
def solve():
    ABC = list(map(int, input().split()))
    if ((ABC[0] == 5) and (ABC[1] == 5) and (ABC[2] == 7))\
            or ((ABC[0] == 5) and (ABC[1] == 7) and (ABC[2] == 5))\
            or ((ABC[0] == 7) and (ABC[1] == 5) and (ABC[2] == 5)):
        print('YES')
        return

    print('NO')


def main():
    solve()


# 実行
main()

File: 042/a.cpp
#include <iostream>
using namespace std;

void solve(){
  int A, B, C;
  cin >> A >> B >> C;
  if( ((A == 5) && (B == 5) && (C == 7)) ||
      ((A == 5) && (B == 7) && (C == 5)) ||
      ((A == 7) && (B == 5) && (C == 5)) )
  { cout << "YES" << endl; return; } 

  cout << "NO" << endl; return;

}

int main(){ 
  solve();
  return 0;
}